docker-compose.development.override.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+
+services:
+  ms: &ms
+    build: .
+    environment:
+      DATABASE_URL: postgres://postgres@db:5432/msf_dev
+
+    volumes:
+      - .:/usr/src/app
```

Comment:
@defektive I know, but there are files stored in the users home directory under `.msf4` where loots and logs are stored. These should lie outside the docker container so you don't loose any data